website/docs/openmp.md

```diff
@@ -0,0 +1,29 @@
+Enable or disable [OpenMP](https://en.wikipedia.org/wiki/OpenMP).
+
+```lua
+openmp "value"
+```
+If no value is set for a configuration, the toolsets default OpenMP option (usually Off) will be performed.
+
+### Parameters ###
+
+`value` is one of:
+
+| Value   | Description                                       |
+|---------|---------------------------------------------------|
+| On      | Turn on OpenMP.                                   |
+| Off     | Turn off OpenMP.                                  |
+
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
+
+Require a version 5.0 alpha 16 or later. Currently only implemented for Visual Studio 2010+.
+
+## Examples ##
+
+```lua
+openmp "On"
```

Comment:
How about

```lua
filter "toolset:msc*"
  openmp "On"
filter "toolset:not msc*"
  buildoptions "-fopenmp"
```

Reply:
Do you have a try that `filter "toolset:not msc*"
  buildoptions "-fopenmp"` works in gcc/clang? `buildoptions "/openmp"` doesn't work in msc.

Reply:
In my sample-project test (https://github.com/Jarod42/premake-sample-projects/tree/OpenMPOld),

I used:

```
filter "toolset:msc*"
		buildoptions "/openmp"
	filter "toolset:not msc*"
		buildoptions "-fopenmp"
```
and my tests pass for visual studio 2019 (I only test that version of visual) as well:

https://github.com/Jarod42/premake-sample-projects/runs/3608755488?check_suite_focus=true

I suggest to continue this discussion in my repo instead of here...

Reply:
See our discussions in https://github.com/Jarod42/premake-sample-projects/commit/65b513d671473923d3a5a964e162ce72bbb1acf3.

Reply:
Marking this resolved, as conversation has moved.